lib/mayaUsd/fileio/primUpdaterManager.h

```diff
@@ -70,8 +74,16 @@ class PrimUpdaterManager : public PXR_NS::TfWeakBase
     MAYAUSD_CORE_PUBLIC
     static PrimUpdaterManager& getInstance();
 
+    /// \brief Verify if there are any data edited as Maya data.
+    MAYAUSD_CORE_PUBLIC
     bool hasPulledPrims() const;
 
+    using PulledPrimPaths = std::vector<std::pair<Ufe::Path, MDagPath>>;
+
+    /// \brief Retrieve the UFE path of the edited USD data and the corresponding path of Maya data.
+    MAYAUSD_CORE_PUBLIC
+    PulledPrimPaths getPulledPrimPaths() const;
+
```

Comment:
Again, the only relevant change is this new function to retrieve the list of known edited-as-Maya root nodes.